src/transaction.rs

```diff
@@ -117,7 +105,7 @@ where
     }
 }
 
-impl<R, C> Iterator for StmtIter<'_, R, C>
+impl<R, C: FirebirdClient> Iterator for StmtIter<'_, R, C>
 where
     R: FromRow,
     C: FirebirdClient,
```

Comment:
```suggestion
impl<R, C> Iterator for StmtIter<'_, R, C>
where
    R: FromRow,
    C: FirebirdClient,
```

The trait bound is repeated